package.json

```diff
@@ -21,6 +21,7 @@
     "angular-material": "1.0.7",
     "angular-messages": "1.5.3",
     "angular-ui-router": "0.2.18",
+    "eslint": "^3.17.1",
```

Comment:
Is this required in the dependencies block? Can we move this to the devDependencies if its not required to run the Application. We are introducing a high severity command injection vulnerability if we keep this here.